ui/package.json

```diff
@@ -70,25 +70,23 @@
         "moment": "2.29.1",
         "moment-timezone": "0.5.33",
         "multer": "1.4.3",
-        "next": "10.2.3",
-        "next-compose-plugins": "2.2.1",
-        "next-routes": "1.4.2",
+        "next": "^12.0.4",
```

Comment:
we decided to use exact versions ( no carrot symbol ) to not break builds on-prem